integration/testdata/helm-render/builds.out.json

```diff
@@ -0,0 +1 @@
+{"builds":[{"imageName":"gcr.io/k8s-skaffold/skaffold-helm","tag":"gcr.io/k8s-skaffold/skaffold-helm:sha256-nonsenselettersandnumbers"}]}
```

Comment:
is this file necessary?

Reply:
this file is used for the integration test as it gets passed to `skaffold render --build-artifacts=`